### PR TITLE
Path Regex Fix

### PR DESCRIPTION
**Pull request by vincentdass**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -65,5 +65,6 @@
   "loc.messages.Initiateddeploymentviakuduserviceforwebapppackage": "Initiated deployment via kudu service for webapp package : %s",
   "loc.messages.MSDeploygeneratedpackageareonlysupportedforWindowsplatform": "MSDeploy generated package are only supported for Windows platform.",
   "loc.messages.UnsupportedinstalledversionfoundforMSDeployversionshouldbealteast3orabove": "Unsupported installed version: %s found for MSDeploy. version should be alteast 3 or above",
-  "loc.messages.UnabletofindthelocationofMSDeployfromregistryonmachineError": "Unable to find the location of MS Deploy from registry on machine (Error : %s)"
+  "loc.messages.UnabletofindthelocationofMSDeployfromregistryonmachineError": "Unable to find the location of MS Deploy from registry on machine (Error : %s)",
+  "loc.messages.PackageDoesNotExist": "Package Path (%s) was specified but it does not exist"
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson]: No package found with specified pattern

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 0
+        "Patch": 1
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/task.json]: No need to increase patch version

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts [hunk 3]
```diff
@@ -36,6 +36,11 @@ async function run() {
         SPN["tenantID"] = endPointAuthCreds.parameters["tenantid"];
         SPN["subscriptionId"] = tl.getEndpointDataParameter(connectedServiceName, 'subscriptionid', true);
 
+        var availableWebPackages = tl.glob(webDeployPkg);
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Cover the following scenarios

1. No package matching the specified pattern --> It should fail
2. Only one package matching specified pattern --> It should pass
3. More than one package found --> It should fail

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts [hunk 4]
```diff
@@ -36,6 +36,11 @@ async function run() {
...
+        if(availableWebPackages.length == 0) {
+            throw new Error(tl.loc('PackageDoesNotExist', webDeployPkg));
+        }
```
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: The task should fail. Done

File: Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson (hunk 1 above)
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson]: Done

File: Tasks/AzureRmWebAppDeployment/task.json (hunk 2 above)
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/task.json]: Done

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts (hunk 3 above)
Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Done

File: Tasks/AzureRmWebAppDeployment/task.json [hunk 5]
```diff
@@ -94,7 +94,7 @@
             "label": "Package or Folder",
             "defaultValue": "",
             "required": true,
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/task.json]: Can we add default value to package input as $(System.DefaultWorkingDirectory)\**\*.zip.

Comment (vincentdass) [inline Tasks/AzureRmWebAppDeployment/task.json]: done

**Pull request merged**